app/views/devise/shared/_links.html.erb

```diff
@@ -2,7 +2,7 @@
   <%= link_to t(".sign_in"), new_session_path(resource_name) %><br />
 <% end -%>
 
-<%- if devise_mapping.registerable? && controller_name != 'registrations' && !Account.global_tenant? && current_account.try(:allow_signup) == "true"  %>
+<%- if devise_mapping.registerable? && controller_name != 'registrations' && !Account.global_tenant? && current_account.try(:allow_signup) == true  %>
```

Comment:
if this is boolean, wouldn't it just be `current_account.try(:allow_signup)` w/o the `== true` part?

Reply:
Yes 🤦‍♂️

Reply:
Resolved in https://github.com/samvera/hyku/pull/1881